front-end/src/routes/index.js

```diff
@@ -1,13 +1,14 @@
 // MainRoutes.jsx
 import React from "react";
 import { Routes, Route } from "react-router-dom";
-import Login from "../pages/login/login.jsx";
-import Header from "../components/header/header.jsx";
+import LoginPage from "../pages/login/login.jsx";
+import SignUpPage from "../pages/signup/signup.jsx";
```

Comment:
sorry, i don't get the point of signup page here. Is not it the same as login page? if you use Slack authentication, all users should be signed up already or is it just a component for future updating events?

Reply:
I was thinking to add manual sign up too. They are both almost ready. Once we can decide about it if we want to add or not.